### ISSUE TITLE
Convert with plugin ni to augeas lens=Ntp.lns

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#1193]
I tried to convert a snippet on the website, but got the error: 

## Used plugins
```
Input:
- Plugin: ini
- Format: ini
Output:
- Plugin: augeas lens=Ntp.lns
- Format: none
```

## Input configuration
```
= /etc/ntp.conf
"#comment[1]" = /etc/ntp.conf, configuration for ntpd; see ntp.conf(5) for help
"#comment[5]" = server ntp.your-provider.example
"#comment[6]" = pool.ntp.org maps to about 1000 low-stratum NTP servers.  Your server will
"#comment[7]" = pick a different set every time it starts up.  Please consider joining the
"#comment[8]" = pool: <http://www.pool.ntp.org/join.html>
server[1] = 0.debian.pool.ntp.org
server[2] = 1.debian.pool.ntp.org
server[3] = 2.debian.pool.ntp.org
server[4] = 3.debian.pool.ntp.org
"#comment[9]" = Access control configuration; see /usr/share/doc/ntp-doc/html/accopt.html for
"#comment[10]" = details.  The web page <http://support.ntp.org/bin/view/Support/AccessRestrictions>
driftfile = /var/lib/ntp/ntp.drift
"#comment[11]" = might also be helpful.
"#comment[12]" = 
"#comment[13]" = "Note that "restrict" applies to both servers and clients, so a configuration"
"#comment[14]" = that might be intended to block requests from certain clients could also end
"#comment[15]" = up blocking replies from your own upstream servers.
"#comment[16]" = By default, exchange time with everybody, but don't allow configuration.
restrict[1] = -4
restrict[2] = default
"#comment[17]" = Local users may interrogate the ntp server more closely.
restrict[3] = 127.0.0.1
"#comment[2]" = Enable this if you want statistics to be logged.
restrict[4] = ::1
"#comment[18]" = Clients from this (example!) subnet have unlimited access, but only if
"#comment[19]" = cryptographically authenticated.
"#comment[20]" = restrict 192.168.123.0 mask 255.255.255.0 notrust
"#comment[21]" = If you want to provide time to your local subnet, change the next line.
"#comment[22]" = (Again, the address is an example only.)
"#comment[23]" = broadcast 192.168.123.255
"#comment[24]" = If you want to listen to time broadcasts on your local subnet, de-comment the
"#comment[25]" = next lines.  Please do this only if you trust everybody on the network!
"#comment[26]" = disable auth
"#comment[3]" = statsdir /var/log/ntpstats/
"#comment[27]" = broadcastclient
statistics/loopstats = 
statistics/peerstats = 
statistics/clockstats = 
filegen[1]/file = loopstats
filegen[1]/type = day
filegen[1]/enable = enable
filegen[2]/file = peerstats
filegen[2]/type = day
filegen[2]/enable = enable
statistics = 
filegen[3]/file = clockstats
filegen[3]/type = day
filegen[3]/enable = enable
server[1]/iburst = 
server[2]/iburst = 
server[3]/iburst = 
server[4]/iburst = 
restrict[1]/action[1] = default
restrict[1]/action[2] = kod
restrict[1]/action[3] = notrap
filegen[1] = loopstats
restrict[1]/action[4] = nomodify
restrict[1]/action[5] = nopeer
restrict[1]/action[6] = noquery
restrict[2]/ipv6 = 
restrict[2]/action[1] = kod
restrict[2]/action[2] = notrap
restrict[2]/action[3] = nomodify
restrict[2]/action[4] = nopeer
restrict[2]/action[5] = noquery
filegen[2] = peerstats
filegen[3] = clockstats
"#comment[4]" = You do need to talk to an NTP server or two (or three).
```

## Last output configuration
```

```

**Comment by markus2330** [comment]
Also import to augeas seems to fail.

**Comment by Namoshek** [comment]
Can you please also try what you are doing on the website using kdb locally?

**Comment by markus2330** [comment]
> Can you please also try what you are doing on the website using kdb locally?

I only did as requested in #1192

But you are right, it seems like it does not roundtrip with ini.

I fixed the augeas error message again: 8a3f665c3113d530230c39f37a8b96a1c7980a98
